django_ace/static/django_ace/widget.js

```diff
@@ -104,6 +106,12 @@
         if (wordwrap == "true") {
             editor.getSession().setUseWrapMode(true);
         }
+        if (!!minlines) {
+            editor.setOption("minLines", minlines);
```

Comment:
It is perfectly happy with string :) and in their [docs](https://github.com/ajaxorg/ace/wiki/Configuring-Ace#renderer-options) there is no mention of type.